lib/fill/index.js

```diff
@@ -0,0 +1,9 @@
+import curryN from '../curryN'
+
+export default curryN(4, function fill(arr, value, start, end) {
```

Comment:
you can set start = 0 and end = arr.length -1 and then use curryN(2, ...) but you won't be able to specify start and end anymore

As a solution, you can use not a value, but an object 
```
{
  value,
  start = 0,
  end = arr.length - 1
}
```

Reply:
It's a part of ES6, we use ES5 only while we have Node LTS 4/6

Reply:
yeah, I mean object as a second parameter and then parse it

But better idea will be to remove start and end at all
It's a different method (like slice, you can use it after in chain)

Reply:
So, should I just remove them?

Reply:
Yeah, and place 2 instead of 4 then